tool/site/client/src/components/dispense/createDispenseNotification.ts

```diff
@@ -50,21 +50,24 @@ export function createDispenseNotification(
     prescriptionOrderMessageHeader,
     [
       dispenseNotificationPatient,
-      ...medicationDispenses
+      ...medicationDispenses,
     ].map(resource => resource.id),
     amendId
   )
 
+  const arrayresources = [
```

Comment:
casing `arrayResources` not sure if this is the best name either, could be something like `resourceArray` or `fhirResources`?